core/include/bufr/encoders/netcdf/Encoder.h

```diff
@@ -0,0 +1,94 @@
+/*
```

Comment:
since we are hosting this, we don't need a license header in every file, can just have a top level license and be done with it.

Reply:
I'm not sure it even needs a string, I think since we are hosting it, just the root LICENSE is sufficient